tests/upgrade/integration/v3/cli/query.go

```diff
@@ -126,3 +128,18 @@ func QueryProposal(container, id string) (govtypesv1.Proposal, error) {
 	}
 	return resp, nil
 }
+
+func QueryFeemarketParams(container string) (feemarkettypes.Params, error) {
```

Comment:
is this query needed?

Reply:
I think they are just used for testing.I'll remove them